README.md

```diff
@@ -1 +1,32 @@
-# node-modules-core-lab
\ No newline at end of file
+# Importing fs, url and path
+
+## Objectives
+
+1. Write a script to import fs, url and path
+1. Invoke methods from core modules fs, url and path
+1. Output results
+
+## Introduction
+
+In this lab, you'll start mastering the three core modules: fs, url and path.
```

Comment:
Beef up the intro - you'll be building a URL transformer that reads a parses a text file to strip away everything from each URL except the domain and protocol...